From 90db57a1740be0ad6ad3efae708899c29cbf30ea Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Thu, 30 Mar 2017 17:04:04 +0300
Subject: [PATCH 1629/5242] MLK-14580 ARM: dts: imx7d-arm2: enable fec1 and
 usdhc nodes

commit  eecf9ccd3c02008bc5fd3bdaecc9beb27c0f2e60 from
https://source.codeaurora.org/external/imx/linux-imx.git

These devices are enabled on imx_4.1.y, enable them on imx_4.9.y as
well.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
index faa5522..1d12619 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2.dts
@@ -194,7 +194,7 @@
 	phy-mode = "rgmii";
 	phy-handle = <&ethphy1>;
 	fsl,magic-packet;
-	status = "disabled";
+	status = "okay";
 
 	mdio {
 		#address-cells = <1>;
@@ -979,7 +979,7 @@
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	vmmc-supply = <&reg_sd1_vmmc>;
-	status = "disabled";
+	status = "okay";
 };
 
 &usdhc2 {
@@ -993,7 +993,7 @@
 	keep-power-in-suspend;
 	enable-sdio-wakeup;
 	vmmc-supply = <&reg_sd2_vmmc>;
-	status = "disabled";
+	status = "okay";
 };
 
 &usdhc3 {
@@ -1007,7 +1007,7 @@
 	bus-width = <8>;
 	non-removable;
 	keep-power-in-suspend;
-	status = "disabled";
+	status = "okay";
 };
 
 &wdog1 {
-- 
1.7.9.5

